specification/devcenter/DevCenter/tspconfig.yaml

```
@@ -0,0 +1,33 @@
+parameters:
+  service-dir:
+    default: "sdk/devcenter"
+emit: [
+  "@azure-tools/typespec-autorest",
+]
+linter:
+  extends: 
+   - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    output-file: "{azure-resource-provider-folder}/Microsoft.DevCenter/{version-status}/{version}/devcenter.json"
+    examples-directory: "examples"     
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Developer.DevCenter"
+    namespace: "{package-dir}"
+    clear-output-folder : true
```

Please add these configs in tspconfig.yaml:
```suggestion
  "@azure-tools/typespec-csharp":
    package-dir: "Azure.Developer.DevCenter"
    namespace: "{package-dir}"
    clear-output-folder : true
    flavor: azure
```